src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -20,30 +20,109 @@
         class="mx-2"
         align="center"
       >
+        <!--        <v-col>-->
```

Comment:
don't know vue -- but is this just a comment (thus probably to be removed) or has some "functional meaning"?

Reply:
sorry, will remove it.